reference/5.1/Microsoft.PowerShell.Core/About/about_Environment_Variables.md

```diff
@@ -23,66 +23,158 @@ other programs. For example, the `WINDIR` environment variable contains the
+type:
 
 ```powershell
-Set-Location Env:
+Get-Item -Path Env:* | Get-Member
 ```
 
-Then, to display the contents of the `Env:` drive, type:
+## Environment variables that store preferences
 
-```powershell
-Get-ChildItem
-```
+PowerShell features can use environment variables to store user preferences.
+These variables work like preference variables, but they are inherited by child
+sessions of the sessions in which they are created. For more information about
+preference variables, see [about_preference_variables](about_Preference_Variables.md).
 
-You can view the environment variables in the `Env:` drive from any other
-PowerShell drive, and you can go into the `Env:` drive to view and change the
-environment variables.
+The environment variables that store preferences include:
+
+- PSExecutionPolicyPreference
+
+  Stores the execution policy set for the current session. This environment
+  variable exists only when you set an execution policy for a single session.
+  You can do this in two different ways.
+
+  - Start a session from the command line using the **ExecutionPolicy**
+    parameter to set the execution policy for the session.
 
-### Environment Variable Objects
+  - Use the `Set-ExecutionPolicy` cmdlet. Use the Scope parameter with
+    a value of "Process".
 
-In PowerShell, each environment variable is represented by an object
-that is an instance of the **System.Collections.DictionaryEntry** class.
+    For more information, see [about_Execution_Policies](about_Execution_Policies.md).
 
-In each **DictionaryEntry** object, the name of the environment variable is the
-dictionary key. The value of the variable is the dictionary value.
+- PSModuleAnalysisCachePath
 
-To display an environment variable in PowerShell, get an object that represents
-the variable, and then display the values of the object properties. When you
-change an environment variable in PowerShell, use the methods that are
-associated with the **DictionaryEntry** object.
+  PowerShell provides control over the file that is used to cache data about a
+  module, such as the commands it exports. The cache is read at startup while
+  searching for a command and is written on a background thread sometime after
+  a module is imported.
 
-To display the properties and methods of the object that represents an
-environment variable in PowerShell, use the `Get-Member` cmdlet. For example,
-to display the methods and properties of all the objects in the `Env:` drive,
-type:
+  Default location of the cache is:
+
+  - `$env:LOCALAPPDATA\Microsoft\Windows\PowerShell`
+
+  The default filename for the cache is `ModuleAnalysisCache`. When you have
+  multiple instances of PowerShell installed, the filename will include a
+  hexadecimal suffix so that there is a a unique filename per installation.
+
+  To change the default location of the cache, set the environment variable
+  before starting PowerShell. Changes to this environment variable only affect
```

Comment:
I think the note about the effect of env vars should be higher up when first talking about env vars.  The key thing is:

1. changes to env vars affects the current process and any new child processes
2. changes to env vars at machine or user scope (Windows only) affects new processes

Reply:
Moved the "Changing" section to the top of the article